rpc/tests/rpc.rs

```diff
@@ -1,77 +1,90 @@
+    use kakarot_rpc_utils::wiremock_utils::setup_rpc_server;
 
     #[tokio::test]
-    async fn when_call_block_number_return_ok() {
-        // Given
-        // Mock config, ethereum light client and starknet light client.
-        let mut starknet_lightclient_mock = config();
+    async fn test_eth_chain_id_is_ok() {
+        let (_, server_handle) = setup_rpc_server().await;
+        let client = reqwest::Client::new();
+        let res = client
+            .post("http://127.0.0.1:3030")
+            .body("{\"jsonrpc\": \"2.0\", \"id\": 1, \"method\": \"eth_chainId\", \"params\": [] }")
+            .header("content-type", "application/json")
+            .send()
+            .await
+            .unwrap();
 
-        // Set expect to testing RPC method
-        starknet_lightclient_mock
-            .expect_block_number()
-            .returning(|| Ok(1));
+        let _has_stop = server_handle.stop().unwrap();
 
-        let kakarot_rpc_mock = KakarotEthRpc::new(Box::new(starknet_lightclient_mock));
+        // check rpc response match our needs
+        // for this request check hexa is KKRT
+    }
 
-        let result_mock = kakarot_rpc_mock.starknet_client.block_number().await;
+    #[tokio::test]
+    async fn test_get_block_number_is_ok() {
+        let (_, server_handle) = setup_rpc_server().await;
+        let client = reqwest::Client::new();
+        let res = client
+            .post("http://127.0.0.1:3030")
+            .body("{\"jsonrpc\": \"2.0\", \"id\": 1, \"method\": \"eth_blockNumber\", \"params\": [] }")
+            .header("content-type", "application/json")
+            .send()
+            .await
+            .unwrap();
 
-        // Then
-        assert_eq!(1, result_mock.unwrap());
+        let _has_stop = server_handle.stop().unwrap();
+
+        // response should return a fixed block id for the moment eg. 19640
     }
 
     #[tokio::test]
-    async fn when_get_code_then_should_return_bytes() {
-        // Given
-        let mut starknet_lightclient_mock = config();
-        let starknet_block_id = StarknetBlockId::Number(1);
-        let ethereum_address = Address::from_slice(&[
-            0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19,
-        ]);
-        let bytes = vec![
-            1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20,
-        ];
-        let bytes_result = Bytes::from(bytes.clone());
-        let bytes_result_value = bytes_result.clone();
+    async fn test_get_code_is_ok() {
+        let (_, server_handle) = setup_rpc_server().await;
+        let client = reqwest::Client::new();
+        let res = client
+            .post("http://127.0.0.1:3030")
+            .body("{\"jsonrpc\": \"2.0\", \"id\": 1, \"method\": \"eth_getCode\", \"params\": [\"0xabde1007dcf45cb509ddde375162399a99880064\", \"latest\"] }")
+            .header("content-type", "application/json")
+            .send()
+            .await
+            .unwrap();
```

Comment:
If we're testing the "absence" of error by unwrapping, it'd be more idiomatic to not unwrap() and assert that the `res.is_ok()` if it is a Result and is Some if it is an Option